lib/terminal/server.go

```diff
@@ -64,6 +65,9 @@ type ServerOpts struct {
 	// RuntimeOpts that is sent before the server is started.
 	ConfigOutput io.Writer `json:"-"`
 
+	// DisableReflect disables gRPC reflection.
+	DisableReflect bool `json:"disable_reflect"`
```

Comment:
Any particular reason to make it opt-out vs opt-in?

Reply:
Opt-out is for the paranoid, opt-in so I need less flags to play with my local server. :)